.coderabbit.yml

```diff
@@ -0,0 +1,25 @@
+language: "en"
+early_access: false
+reviews:
+  high_level_summary: true
+  poem: true
+  review_status: true
+  collapse_walkthrough: false
+  path_filters:
+    - "!**/*.json"
+    - "!op-bindings/bindings/**"
+  path_instructions:
+    - path: "**.sol"
+      instructions: "Focus on the following areas:
+        - Security
+        - Identifying test cases which are lacking
+        - Removing unnecessary code
+      "
```

Comment:
I included a single path and instructions field just to start testing it out. I really don't want to bikeshed over it here. Feel free to submit a suggestion if you want it included, or just create a follow up PR.